<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepModeWindow</class>
 <widget class="QWidget" name="StepModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="nextStepButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>NextStep</string>
   </property>
  </widget>
  <widget class="QLabel" name="rejectedLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rejected                     </string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>9</y>
     <width>151</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>160</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>160</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
